src/tests/desktop-agent.spec.js

```diff
@@ -34,17 +41,39 @@ describe("Desktop Agent", function() {
       it("is defined", function() {
         expect(desktopAgent.broadcast).toBeDefined();
       });
+      it("should not return a value", function() {
+        let context = {};
+        let result = desktopAgent.broadcast(context);
+        expect(result).toBeNull();
```

Comment:
Same as above, a valid result could be undefined as well since null is an explicitly set state. 

Maybe we could use `toBeFalsy()`?

Reply:
Will replace with `toBeFalsy()` and update the implementation